client/my-sites/plans-features-main/index.jsx

```diff
@@ -268,16 +268,20 @@ class PlansFeaturesMain extends Component {
 	}
 
 	render() {
-		const { site } = this.props;
+		const { site, isInSignup } = this.props;
```

Comment:
Hmm, okay. But isn't this a permanent thing not to display FAQ in signup? I can understand your concern but I still think this one is fine. Like: is in signup? alright then, but hide FAQ. Otherwise show.

But will do.